engines/restic.go

```diff
@@ -211,6 +212,13 @@ func (r *ResticEngine) forget() (err error) {
 	v := r.Volume
 
 	snapshots, err := r.snapshots()
+	if err != nil {
+		return
+	}
+	if len(snapshots) == 0 {
+		err = errors.New("No snapshots found but bucket should contains at least current backup")
```

Comment:
please use log.Errorf() or log.Fatalf()

Reply:
log.Errorf() just generate a message on stderr, here we want to create a error var.